src/main/java/ru/yandex/qatools/allure/command/ReportGenerate.java

```diff
@@ -89,4 +89,21 @@ protected String getLoggerConfigurationArgument() {
         return String.format("-Dorg.slf4j.simpleLogger.defaultLogLevel=%s",
                 isQuiet() || !isVerbose() ? "error" : "debug");
     }
+
+    private String getClassPathArgument() {
+        String version = getConfig().getCurrentVersion();
+        String separator = System.getProperty("path.separator");
+        Path configPath = getConfigPath().getParent().toAbsolutePath();
+        Path bundleJarPath = getExecutablePath(version);
+        String pluginsPath = getPluginsPath().toAbsolutePath() + "/*";
```

Comment:
okeey, I am a bit surprised that this works on windows - I thought that Windows executable should be `%s/bin/java.exe`